test/microapps-ui/microapps-ui-main.check.pools.test.ts

```diff
@@ -0,0 +1,217 @@
+/*
+ *
+ * @group microappsUI
+ */
+import { jest } from "@jest/globals";
+import { ApiPromise, Keyring } from "@polkadot/api";
+import { getApi, initApi, getMangataInstance } from "../../utils/api";
+import {
+  BN_TEN_THOUSAND,
+  BN_THOUSAND,
+  MangataInstance,
+} from "@mangata-finance/sdk";
+import { Node } from "../../utils/Framework/Node/Node";
+import { WebDriver } from "selenium-webdriver";
+import { DriverBuilder } from "../../utils/frontend/utils/Driver";
+import { Sidebar } from "../../utils/frontend/microapps-pages/Sidebar";
+import { LiqPools } from "../../utils/frontend/microapps-pages/LiqPools";
+import {
+  addExtraLogs,
+  importPolkadotExtension,
+} from "../../utils/frontend/utils/Helper";
+import {
+  DUMMY_POOL_ASSET_ID,
+  FIVE_MIN,
+  KSM_ASSET_ID,
+  MGA_ASSET_ID,
+} from "../../utils/Constants";
+import { ApiContext } from "../../utils/Framework/XcmHelper";
+import XcmNetworks from "../../utils/Framework/XcmNetworks";
+import { connectVertical } from "@acala-network/chopsticks";
+import StashServiceMockSingleton from "../../utils/stashServiceMockSingleton";
+import { AssetId } from "../../utils/ChainSpecs";
+import { getEnvironmentRequiredVars } from "../../utils/utils";
+import { AssetWallet, User } from "../../utils/User";
+import {
+  connectWallet,
+  setupPage,
+} from "../../utils/frontend/microapps-utils/Handlers";
+import "jest-extended";
+
+jest.setTimeout(FIVE_MIN);
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+let driver: WebDriver;
+let testUser1: User;
+
+const acc_name = "acc_automation";
+const userAddress = "5CfLmpjCJu41g3cpZVoiH7MSrSppgVVVC3xq23iy9dZrW2HR";
+describe("Microapps UI liq pools tests", () => {
+  let kusama: ApiContext;
+  let mangata: ApiContext;
+  let sdk: MangataInstance;
+  let api: ApiPromise;
+
+  beforeAll(async () => {
+    kusama = await XcmNetworks.kusama({ localPort: 9944 });
+    mangata = await XcmNetworks.mangata({ localPort: 9946 });
+    await connectVertical(kusama.chain, mangata.chain);
+    StashServiceMockSingleton.getInstance().startMock();
```

Comment:
These are non-chopstick tests, data about pools is taken from stash service (or somewhere else) so we don't need these chops setups. This test can nicely run when pointed to prod or any env. Chops would be needed only if we would like to add some extra pools by ourselves.

Reply:
Changed